docs/extra_fields.md

```diff
@@ -0,0 +1,64 @@
+# Instructions for Adding Fields under "extras:"
+
+These fields provide essential information about the item, making it easier for users to understand its purpose and where to find more details.
+The `extras` section in the YAML file allows you to include additional information about your project. You can add various fields to provide essential details, making it easier for users to understand its purpose and where to find more information. You can add various fields such as `description`, `website`, `repo_url`, `logo`, and `category`.
+
+These fields provide essential information about the item, making it easier for users to understand its purpose and where to find more details.
```

Comment:
This line seems to be a duplicate, it appears in the previous paragraph.

Reply:
Good catch, but deleted the file